specs/phase0/validator.md

```diff
@@ -612,23 +612,22 @@ Because Phase 0 does not have shards and thus does not have Shard Committees, th
 
 * Remain subscribed to `SUBNETS_PER_NODE` for `EPOCHS_PER_SUBNET_SUBSCRIPTION` epochs.
 * Maintain advertisement of the selected subnets in their node's ENR `attnets` entry by setting the selected `subnet_id` bits to `True` (e.g. `ENR["attnets"][subnet_id] = True`) for all persistent attestation subnets.
-* Select these subnets based on their node-id as specified by the following `compute_subnets(node_id,epoch)` function.
+* Select these subnets based on their node-id as specified by the following `compute_subscribed_subnets(node_id,epoch)` function.
 
 ```python
-def compute_subnet(node_id: int, epoch: Epoch, index: int) -> int:
+def compute_subscribed_subnet(node_id: int, epoch: Epoch, index: int) -> int:
     node_id_prefix = node_id >> (256 - ATTESTATION_SUBNET_PREFIX_BITS)
-    permutation_seed = hash(uint_to_bytes(epoch // EPOCHS_PER_SUBNET_SUBSCRIPTION))
+    node_offset = node_id % EPOCHS_PER_SUBNET_SUBSCRIPTION
```

Comment:
This means that the same 1/256 group always changes at the same time, right? (but to different places)

I had imagined the periods being independent and a node's period spanning [256, 512) such that there weren't defined groupings that churn together. This is certainly simpler

But I do want to quickly bring up if there is any sort of security issue in having persistent 1/256 groupings (even though independent destinations)

I think not, but it semi smells funny

Reply:
misc intervals don't seem too bad to me. we can easily model the high and low end of probabilistic size of subnets

that said, a shift in pre-built chunks like this is probably fine